lib/matplotlib/widgets.py

```diff
@@ -255,7 +255,7 @@ class Slider(AxesWidget):
     def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
                  closedmin=True, closedmax=True, slidermin=None,
                  slidermax=None, dragging=True, valstep=None,
-                 orientation='horizontal', **kwargs):
+                 orientation='horizontal', initcolor='r', **kwargs):
```

Comment:
```suggestion
                 orientation='horizontal', *, initcolor='r', **kwargs):
```
We're nowadays more conservative and add new keyword arguments as keyword-only most of the time.